src/lib/relay/QueryRenderers.tsx

```diff
@@ -302,6 +303,31 @@ export const CollectionRenderer: React.SFC<CollectionRendererProps> = ({ collect
   />
 )
 
+export const CollectionFullFeaturedArtistListRenderer: React.SFC<CollectionRendererProps> = ({
+  collectionID,
+  render,
+}) => (
+  <QueryRenderer<QueryRenderersCollectionFullFeaturedArtistListQuery>
+    environment={environment}
+    query={graphql`
+      query QueryRenderersCollectionFullFeaturedArtistListQuery($collectionID: String!, $screenWidth: Int) {
+        collection: marketingCollection(slug: $collectionID) {
+          ...FullFeaturedArtistList_collection @arguments(screenWidth: $screenWidth)
```

Comment:
Do we need to pass `screenWidth` here? It doesn't seem like it's being used (since the images are queried for from the `ArtistListItem` component).